app/static/src/scss/includes/_browse.scss

```diff
@@ -394,7 +394,7 @@
   }
 
   .govuk-summary-list__row {
-    display: flex;
```

Comment:
just suggestion, not sure if this is applied on base level, if you can copy the css and put in _record.scss and undo _browse.scss changes, can that not work ?